Tests/JobsTests/JobsTests.swift

```diff
@@ -26,7 +26,8 @@ final class JobsTests: XCTestCase {
         let app = try self.startServer()
         defer { app.shutdown() }
         app.jobs.schedule(Cleanup()).hourly().at(30)
-        XCTAssertEqual(app.make(JobsConfiguration.self).scheduledStorage.count, 1)
+        app.jobs.schedule(Cleanup(), at: Date() + 5)
```

Comment:
Could we use the schedule builder for one-off jobs?
```swift
app.jobs.schedule(Cleanup()).at(Date() + 5)
```
It would be nice to not need two separate call structures depending on what you're trying to do.

Reply:
Agreed